packages/pods-core/src/builtin/OrbitPod.ts

```diff
@@ -0,0 +1,295 @@
+import {
+  Conflict,
+  ImportPod,
+  ImportPodConfig,
+  ImportPodPlantOpts,
+} from "../basev3";
+import { JSONSchemaType } from "ajv";
+import { ConflictHandler, PodUtils } from "../utils";
+import {
+  DendronError,
+  DEngineClient,
+  DVault,
+  ERROR_SEVERITY,
+  getSlugger,
+  NoteProps,
+  NoteUtils,
+  stringifyError,
+  Time,
+} from "@dendronhq/common-all";
+import axios from "axios";
+import _ from "lodash";
+
+const ID = "dendron.orbit";
+
+type OrbitImportPodCustomOpts = {
+  /**
+   * orbit workspace slug
+   */
+  workspaceSlug: string;
+  /**
+   * orbit person access token
+   */
+  token: string;
+};
+
+export enum MergeConflictOptions {
```

Comment:
Skipped the option to Overwrite remote value with local value because the orbit API is not updating contact details

Reply:
for `MergeConflictOptions`, since this is something we want pods to support in general, can we move it to `src/types.ts` instead? the pods themselves should indicate if certain fields aren't supported